fetching the integration version from the pyproject (#106)
port-labs/ocean: changelog/3.improvement.md

> Changed the publish integration workflow to get the integration version from the pyproject.toml of that integration instead of the spec.yml

Reviewer: ```suggestion
Changed the publish integration workflow to get the integration version from the `pyproject.toml` file of the integration and not from the `spec.yml` file
```
